docs/release-notes/release-notes-0.16.0.md

```diff
@@ -66,6 +66,7 @@ minimum version needed to build the project.
 
 * [The `golangci-lint` tool was updated to
   `v1.46.2`](https://github.com/lightningnetwork/lnd/pull/6731)
+* Tests in `htlcswitch` will now clean up the temporary resources they create.
```

Comment:
You need to link to the PR for the CI check to pick up on it.

Reply:
Cheers, added a link and also fixed the spacing.